.github/workflows/release.yaml

```diff
@@ -114,10 +131,7 @@ jobs:
 
   build-and-publish-snap:
     runs-on: ubuntu-latest
-
-    # Skip running workflow on forks
-    if: github.repository_owner == 'prql'
```

Comment:
Sorry to keep adding comments. I do think we need this — otherwise the build fails on a fork...

Reply:
I dont know why you care about it. Other release jobs basically fail too, is that a problem?

Reply:
Hmmm, good point. 

I tried to look at the history of this; I think it inherited from https://github.com/PRQL/prql/blob/40ddf29b588f70b8dda3bf9aed57d0e017e7e973/.github/workflows/build-web.yaml#L26 — where it is needed, since that workflow regularly runs on forks.

Agree re removing, we can see if something breaks...

Reply:
Happy to see whether something breaks, don't want to slow you down. But on thinking about it again — surely every push to a fork *is* going to fail — should we have `github.repository_owner == 'prql'` as a condition on https://github.com/PRQL/prql/pull/3181/files#diff-e3cb659af0d95ff75113e6a8678a6ac089b653b083d316c3f44b4da8086b59d2R183 ?

Reply:
if `dry-run: true`, the workflow will not fail on forks as no secrets are used.
So we don't need to check the repository owner.

![image](https://github.com/PRQL/prql/assets/50911393/f6d3d10f-7266-4cd9-a4c6-847ee405bd61)